src/foam/u2/layout/Card.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'Card',
+  extends: 'foam.u2.borders.CardBorder',
+
+  properties: [
+    {
+      class: 'Int',
+      name: 'def',
```

Comment:
Can you make these a little more descriptive and/or add documentation?
cols - this is the default value for all sizes
mdCols
lgCols
smCols
etc...

Reply:
Yeah I think we shouldn't use such short names, just use full words, it would be more consistent with the rest of foam.

Reply:
Done.